frontend/src/components/AlertForm.js

```diff
@@ -1,36 +1,22 @@
 //import React, { useEffect, useState } from 'react';
-import React, { useState } from 'react';
+import React from 'react';
 
 export default function AlertForm() {
-    const [inputValue, setInputValue] = useState('');
-    const handleInput = (e) => {
-        const formattedPhoneNumber = formatPhoneNumber(e.target.value);
-        setInputValue(formattedPhoneNumber);
-    };
-    
     return (
         <form>
-            <label>
-                Email:
-                <input type="email" name="email" />
-            </label>
-            <label>
-                Phone Number:
-                <input onChange={e => handleInput(e)} value={inputValue} />
-            </label>
-            <input type="submit" value="Sign up" />
+            <div className="form-group mt-3">
+                <label for="userEmail" className="text-muted">Email</label>
+                <input type="email" className="form-control" id="email" aria-describedby="emailForm" placeholder="tommytrojan@usc.edu" required/>
+            </div>
+            <div className="form-check">
+                <input type="checkbox" className="form-check-input" id="emailCheckbox" defaultChecked/>
+                <small className="form-text text-muted">Receive email notifications too</small>
+            </div>
+            <div className="form-group mt-3">
+                <label for="userPhone" className="text-muted">Phone</label>
+                <input type="tel" className="form-control" id="tel" aria-describedby="phoneForm" placeholder="123-456-7890" pattern="^\d{4}-\d{3}-\d{4}$" />
```

Comment:
Very cool that we have validation for these fields, but I wonder if it is too strict 😅 

I manually entered a phone number in the format "1234567890" and it gave me an error because it didn't contain dashes. Could we loosen this up a bit somehow? (i.e. to include formats like "1234567890" or "(123) 456-7890"

Reply:
You might be able to use the function you had there before, but this isn't related to styling of the form though so I think its fine if we handle this in a future PR.

Reply:
One important note is that we must send the number to the backend in the format of `+1xxxxxxx`